underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -159,6 +165,26 @@ public SwiftUnderFileSystem(AlluxioURI uri) {
       container.create();
     }
     mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
+
+    // Assume the Swift user name has 1-1 mapping to Alluxio username.
+    mAccountOwner = Configuration.get(PropertyKey.SWIFT_USER_KEY);
+    short mode = (short) 0;
+    // If there is any container ACL for the Swift user, translates it to Alluxio permission.
+    if (container.getContainerReadPermission().contains(mAccountOwner)
```

Comment:
@ifcharming Do you think it makes sense to at least split the ACL string using delimiter "," and checking if user or * appears as one of the tokens. String.contains might give access to user 'bob' when actually 'bobby' had access.